apigw-websocket-mapping-template-authorizer/src/sendmessage.mjs

```diff
@@ -0,0 +1,45 @@
+        import { DynamoDBClient, ScanCommand, DeleteItemCommand } from "@aws-sdk/client-dynamodb";
+        import { ApiGatewayManagementApiClient, PostToConnectionCommand } from "@aws-sdk/client-apigatewaymanagementapi";
+        const ddbClient = new DynamoDBClient({ region: process.env.AWS_REGION });
+
+        export const handler = async (event) => {
+            let connectionData;
+            console.log("event received:", event); //check the event received by Lambda
```

Comment:
CWE-117 - Log injection High
We detected that unsanitized user data is being written to the logs. Unsanitized data can inject malicious content. To increase the security of your code, sanitize your inputs before logging them.